flink-core/src/test/java/org/apache/flink/api/common/io/FileInputFormatTest.java

```diff
@@ -18,24 +18,20 @@
 
 package org.apache.flink.api.common.io;
 
-import java.io.BufferedOutputStream;
-import java.io.BufferedWriter;
-import java.io.File;
-import java.io.FileOutputStream;
-import java.io.FileWriter;
-import java.io.IOException;
+import java.io.*;
```

Comment:
As per our coding guidelines, star imports are not allowed: "Do not use wildcard imports in the core files. They can cause problems when adding to the code and in some cases even during refactoring. Exceptions are the Tuple classes, Tuple-related utilities, and Flink user programs, when importing operators/functions. Tests are a special case of the user programs." http://flink.apache.org/coding_guidelines.html